components/script/dom/bluetoothremotegattserver.rs

```diff
@@ -46,6 +46,34 @@ impl BluetoothRemoteGATTServer {
     fn get_bluetooth_thread(&self) -> IpcSender<BluetoothRequest> {
         self.global().as_window().bluetooth_thread()
     }
+
+    pub fn set_connected(&self, connected: bool) {
+        self.connected.set(connected);
+    }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#garbage-collect-the-connection
+    #[allow(unrooted_must_root)]
```

Comment:
Without this the following error shows up for [line 61](https://github.com/servo/servo/pull/14612/files/3360085897534f6f9dfb9fa041c1e15db5939586#diff-1dbe29f87740f5aec93f37adbecace6cR61):
```
error: Expression of type std::collections::hash_map::Iter<'_, std::string::String, dom::bindings::js::MutJS<dom::bluetoothdevice::BluetoothDevice>> must be rooted, #[deny(unrooted_must_root)] on by default
  --> /home/zakorgy/servo/components/script/dom/bluetoothremotegattserver.rs:60:9
   |
60 |         for (id, device) in context.get_device_map().borrow().iter() {
   |         ^
```
The problem is, that `device` should be rooted here, but i didn't find another way to solve this correctly.